spark/src/main/java/org/apache/zeppelin/spark/SparkVersion.java

```diff
@@ -34,10 +34,10 @@
   public static final SparkVersion SPARK_1_6_0 = SparkVersion.fromVersionString("1.6.0");
 
   public static final SparkVersion SPARK_2_0_0 = SparkVersion.fromVersionString("2.0.0");
-  public static final SparkVersion SPARK_2_2_0 = SparkVersion.fromVersionString("2.2.0");
+  public static final SparkVersion SPARK_2_3_0 = SparkVersion.fromVersionString("2.3.0");
 
   public static final SparkVersion MIN_SUPPORTED_VERSION =  SPARK_1_0_0;
-  public static final SparkVersion UNSUPPORTED_FUTURE_VERSION = SPARK_2_2_0;
+  public static final SparkVersion UNSUPPORTED_FUTURE_VERSION = SPARK_2_3_0;
```

Comment:
This is due to historical reason that zeppelin will check the supported version. If user want to try some spark version that is not officially supported by zeppelin, he can set `zeppelin.spark.enableSupportedVersionCheck` to `false`.